packages/sirius-web/frontend/sirius-web-application/src/views/library-browser/EditLibraryView.tsx

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { SelectionContextProvider, Workbench } from '@eclipse-sirius/sirius-components-core';
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { makeStyles } from 'tss-react/mui';
+import { EditProjectNavbar } from '../edit-project/EditProjectNavbar/EditProjectNavbar';
+import { EditLibraryViewParams, EditLibraryViewState } from './EditLibraryView.types';
+import { useLibrary } from './useLibrary';
+
+const useEditLibraryViewStyles = makeStyles()(() => ({
+  editProjectView: {
+    display: 'grid',
+    gridTemplateRows: 'min-content min-content minmax(0, 1fr)',
+    gridTemplateColumns: '1fr',
+    height: '100vh',
+    width: '100vw',
+  },
+}));
+
+export const EditLibraryView = () => {
```

Comment:
Library should be immutable, if there's one thing we don't we it is to edit them

Reply:
Done